rustls/src/client/mod.rs

```diff
@@ -433,13 +433,7 @@ impl ClientSessionImpl {
     }
 
     pub fn find_cipher_suite(&self, suite: CipherSuite) -> Option<&'static SupportedCipherSuite> {
-        for scs in &self.config.ciphersuites {
-            if scs.suite == suite {
-                return Some(scs);
-            }
-        }
-
-        None
+        self.config.ciphersuites.iter().copied().find(|&scs| scs.suite == suite)
```

Comment:
This seems unrelated to the other changes. If the goal is to make this the same as some other code, maybe they could share an actual implementation?

Reply:
It wouldn't be right to add a dependency on `ClientSessionImpl` to the `ClientSessionValue`.

I don't think it's strictly unrelated because it helps us understand that the logic is the same. You are right that this change isn't strictly needed, so I could revert this change, if you'd prefer.